tests/wot_coap_generation/config/wot_td/.coap_affordances.json

```diff
@@ -3,16 +3,16 @@
     },
     "events": {
     },
-    "actions": [
-        {
-            "name": "toggle",
-            "url": "/led/toggle",
-            "handler": {
-                "file": "toggle.c",
-                "name": "_led_toggle_handler",
-                "multipleUsage": false
-            },
-            "method": "POST",
+    "actions": {
+        "toggle": {
+            "forms": [
```

Comment:
Why do you want to use an array here?

Reply:
If I am not mistaken it is actually defined in the specification as an array (see https://www.w3.org/TR/wot-thing-description/#interactionaffordance). I also thought it could be an advantage if the definition of the affordances for the script is similar to the actual thing description when it comes both to definition and validation.

Reply:
> If I am not mistaken it is actually defined in the specification as an array (see https://www.w3.org/TR/wot-thing-description/#interactionaffordance). I also thought it could be an advantage if the format of the affordances for the script is similar to the actual thing description when it comes both to definition and validation.

You are right. Now I am a bit confused, because one interaction affordance can have multiple endpoints.